addons/common/functions/fnc_checkFiles.sqf

```diff
@@ -39,29 +39,47 @@ if ([_cbaRequiredAr, _cbaVersionAr] call cba_versioning_fnc_version_compare) the
 
 //private _addons = activatedAddons; // broken with High-Command module, see #2134
 private _addons = (cba_common_addons select {(_x select [0,4]) == "ace_"}) apply {toLower _x};
-
+private _oldAddons = [];
+private _oldSources = [];
 private _oldCompats = [];
 {
-    if (getText (configFile >> "CfgPatches" >> _x >> "versionStr") != _version) then {
-        private _errorMsg = format ["File %1.pbo is outdated.", _x];
-
-        ERROR(_errorMsg);
+    private _addonCfg = configFile >> "CfgPatches" >> _x;
+    private _addonVersion = getText (_addonCfg >> "versionStr");
+    if (_addonVersion != _mainVersion) then {
+        private _addonSource = configSourceMod _addonCfg;
+        _oldSources pushBackUnique _addonSource;
+        call FUNC(checkFiles_diagnoseACE);
 
         if ((_x select [0, 10]) != "ace_compat") then {
             if (hasInterface) then {
-                ["[ACE] ERROR", _errorMsg, {findDisplay 46 closeDisplay 0}] call FUNC(errorMessage);
+                _oldAddons pushBack _x;
             };
         } else {
-            _oldCompats pushBack _x;  // Don't block game if it's just an old compat pbo
+            _oldCompats pushBack [_x, _addonVersion];  // Don't block game if it's just an old compat pbo
```

Comment:
```suggestion
            _oldCompats pushBack [_x, _addonVersion, _addonSource];  // Don't block game if it's just an old compat pbo
```
I think? not sure but looks like line 78 uses it?

Reply:
Forgot to remove after testing. Will just edit line 78.